usecases/objects/merge.go

```diff
@@ -80,6 +80,10 @@ func (m *Manager) MergeObject(ctx context.Context, principal *models.Principal,
 
 func (m *Manager) retrievePreviousAndValidateMergeObject(ctx context.Context, principal *models.Principal,
 	id strfmt.UUID, updated *models.Object) (*search.Result, error) {
+	if updated == nil {
+		return nil, fmt.Errorf("object to update not provided in request")
+	}
```

Comment:
I had a nil pointer panic here when I ran a malformed curl request. This happened only twice and I have not been able to reproduce, but now that we know this *can* happen, here is the fix :)